backends/dpdk/dpdkUtils.cpp

```diff
@@ -75,6 +75,24 @@ bool isEightBitAligned(const IR::Expression *e) {
     return true;
 }
 
+bool isLargeFieldOperand(const IR::Expression *e) {
+    auto expr = e;
+    if (auto base = e->to<IR::Cast>()) expr = base->expr;
+    if (auto type = expr->type->to<IR::Type_Bits>()) {
+        auto size = type->width_bits();
+        if (size > 64) return true;
+    }
+    return false;
+}
+
+bool isHeader(const IR::Expression *expr) {
```

Comment:
Maybe it should be called "isInsideHeader"

Reply:
Will update